conf/application.conf

```diff
@@ -153,6 +153,8 @@ microservice {
   }
 }
 
+cid-client-id = "cid-client-id"
```

Comment:
Should we house this in a container that explains what it is? 

`consumer-client-ids` {
  moj-client-id = "moj-client-id"
}

And the ID is the ID of the consumer (MOJ) in this case. Therefore cid-client-id is not accurate as CID is the recipient not to whom the ID relates.